symmetricWorker.js

```diff
@@ -4,76 +4,82 @@ var _ = require('underscore')
 var app = require('express')()
 var server = require('http').Server(app)
 var io = require('socket.io')(server)
+const rethink = require('rethinkdb')
+const defaultConnection = config.get('defaultConnection')
+const symmetricWorker = config.get('symmetricWorker')
+const pino = require('pino')
+
+const waitingThreshold = config.get('waitingThreshold')
```

Comment:
add to symmetricWorker option

Reply:
requested changes are addressed in new commit ***'requested changes in review'*** .
due to changes in config, necessary changes are also made in this file.